packages/openactive-broker-microservice/README.md

```diff
@@ -294,3 +311,548 @@ Clients credentials can be defined as follows:
+
+Some of the tests in [Integration Tests](../openactive-integration-tests/) use this in order to run checks against the Dataset Site JSON.
+
+Here is an example:
+
+```json
+{
+  "@context": [
+    "https://schema.org/",
+    "https://openactive.io/"
+  ],
+  "@type": "Dataset",
+  "@id": "https://acme-fitness.org/api/",
+  "url": "https://acme-fitness.org/api/",
+  "name": "Acme Fitness",
+  "accessService": {
+    "@type": "WebAPI",
+    "name": "Open Booking API",
+    // ...etc
+  },
+  // ...
+}
+```
+
+#### `DELETE /opportunity-cache`
+
+**Response Type**: 🕳️ Empty
+
+Deletes all of Broker Microservice's cached data about harvested [Opportunities](https://openactive.io/open-booking-api/EditorsDraft/#dfn-opportunity).
+
+This is used by [Integration Tests](../openactive-integration-tests/) to reset Broker Microservice's cache in between runs of Test Suite in [Controlled mode](../openactive-integration-tests/README.md#userandomopportunities).
+This is necessary as the [Booking System](#booking-system-under-test) deletes the entire Test Dataset in between runs of Test Suite in Controlled mode ([Test Interface docs](https://openactive.io/test-interface/#delete-test-interfacedatasetstestdatasetidentifier)).
+
+#### `POST /opportunity-listeners/:id`
+
+**Request params**:
+
+* `id`: The ID of the [Child Opportunity](#orphans) to listen for.
+
+**Response Type**: 🕳️ Empty
+
+Create an [Opportunity Listener](#two-phase-listeners) to listen for updates to the specified Child Opportunity.
+
+#### `GET /opportunity-listeners/:id`
+
+**Request params**:
+
+* `id`: The ID of the [Child Opportunity](#orphans) that is already being listened for.
+
+**Response Type**: 🤖 JSON
+
+Must be called after [`POST /opportunity-listeners/:id`](#post-opportunity-listenersid).
+
+If and when the specified Child Opportunity is updated, this will return the updated Opportunity. This invokes the **Get** phase of the [Opportunity Listener](#two-phase-listeners).
+
+Either this endpoint will:
+
+* If an update was found:
+    * return the updated Opportunity.
+* If no update is found:
+    * eventually timeout.
+
+### `POST /order-listeners/:type/:bookingPartnerIdentifier/:uuid`
+
+**Request params**:
+
+* `type`: `orders` or `order-proposals`.
+* `bookingPartnerIdentifier`: Identifier of the Booking Partner whose [Orders Feed](https://openactive.io/open-booking-api/EditorsDraft/#dfn-orders-feed) or OrderProposals Feed to listen to. This is the key of the Booking Partner in the [`bookingPartners` configuration property](#bookingpartners) e.g. `primary`.
+* `uuid`: UUID of the Order or OrderProposal to listen for.
+
+**Response Type**: 🤖 JSON
+
+Create an [Order Listener](#two-phase-listeners) to listen for updates to the specified Order or Order Proposal.
+
+The response contains info that is useful to a human user for debugging, such as the `startingFeedPage`, which is the page of the feed that Broker Microservice will start listening from.
```

Comment:
Should a copy of an example response be included here, as with others above?

Reply:
Done